config/locales/en.yml

```diff
@@ -1351,6 +1346,27 @@ en:
       clear_all_filters: 'Clear filters'
       remove_filter_hidden: 'Remove this filter'
       return_to_results: Return to results
+  subscription_mailer:
+    confirmation:
+      frequency:
+        daily: at the end of the day (at approx 3 pm)
+        weekly: weekly (at approx 6 pm on friday evening)
+      next_steps: >-
+        You’ll receive a job alert %{frequency} if more than one job matching your criteria is listed.
```

Comment:
Is this true? I thought it was if more than zero jobs (at least one job).
```
vacancies = vacancies_for_subscription(subscription)
      next if vacancies.blank?
```

Reply:
What you say is correct, updated the text

Reply:
> You’ll receive a job alert %{frequency} when a job alert matching your criteria is listed.

I prefer `if` to `when` because `when` refers to a frequency. I'll ask Valentine for a thumbs up.